Fixes warnings like:
/var/tmp/portage/sys-freebsd/freebsd-lib-9.0-r2/work/lib/libopie/../../contrib/opie/libopie/randomchallenge.c:43:5:
warning: incompatible implicit declaration of built-in function ‘strcpy’ 

diff -uNr contrib.old/opie/libopie/accessfile.c contrib/opie/libopie/accessfile.c
--- contrib.old/opie/libopie/accessfile.c	2012-05-25 08:54:33.000000000 -0400
+++ contrib/opie/libopie/accessfile.c	2012-05-25 09:12:34.000000000 -0400
@@ -46,6 +46,7 @@
 #ifdef DEBUG
 #include <syslog.h>
 #endif /* DEBUG */
+#include <ctype.h>
 
 #include "opie.h"
 
diff -uNr contrib.old/opie/libopie/generator.c contrib/opie/libopie/generator.c
--- contrib.old/opie/libopie/generator.c	2012-05-25 08:54:33.000000000 -0400
+++ contrib/opie/libopie/generator.c	2012-05-25 08:58:10.000000000 -0400
@@ -50,6 +50,7 @@
 #include <syslog.h>
 #endif /* DEBUG */
 #include "opie.h"
+#include <stdio.h>
 
 static char *algids[] = { NULL, NULL, NULL, "sha1", "md4", "md5" };
 
diff -uNr contrib.old/opie/libopie/lookup.c contrib/opie/libopie/lookup.c
--- contrib.old/opie/libopie/lookup.c	2012-05-25 08:54:33.000000000 -0400
+++ contrib/opie/libopie/lookup.c	2012-05-25 09:15:08.000000000 -0400
@@ -14,6 +14,7 @@
 #include "opie_cfg.h"
 #include <stdio.h>
 #include "opie.h"
+#include <string.h>
 
 int opielookup FUNCTION((opie, principal), struct opie *opie AND char *principal)
 {
diff -uNr contrib.old/opie/libopie/newseed.c contrib/opie/libopie/newseed.c
--- contrib.old/opie/libopie/newseed.c	2012-05-25 08:54:33.000000000 -0400
+++ contrib/opie/libopie/newseed.c	2012-05-25 09:14:37.000000000 -0400
@@ -36,6 +36,9 @@
 #include <syslog.h>
 #endif /* DEBUG */
 #include "opie.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
 
 int opienewseed FUNCTION((seed), char *seed)
 {
diff -uNr contrib.old/opie/libopie/parsechallenge.c contrib/opie/libopie/parsechallenge.c
--- contrib.old/opie/libopie/parsechallenge.c	2012-05-25 08:54:33.000000000 -0400
+++ contrib/opie/libopie/parsechallenge.c	2012-05-25 09:15:54.000000000 -0400
@@ -20,6 +20,8 @@
 #include <string.h>
 #endif /* HAVE_STRING_H */
 #include "opie.h"
+#include <stdlib.h>
+#include <ctype.h>
 
 struct algorithm {
   char *name;
diff -uNr contrib.old/opie/libopie/passwd.c contrib/opie/libopie/passwd.c
--- contrib.old/opie/libopie/passwd.c	2012-05-25 08:54:33.000000000 -0400
+++ contrib/opie/libopie/passwd.c	2012-05-25 09:13:45.000000000 -0400
@@ -22,6 +22,7 @@
 
 #include "opie_cfg.h"
 #include "opie.h"
+#include <string.h>
 
 int opiepasswd FUNCTION((old, flags, principal, n, seed, ks), struct opie *old AND int flags AND char *principal AND int n AND char *seed AND char *ks)
 {
diff -uNr contrib.old/opie/libopie/randomchallenge.c contrib/opie/libopie/randomchallenge.c
--- contrib.old/opie/libopie/randomchallenge.c	2012-05-25 08:54:33.000000000 -0400
+++ contrib/opie/libopie/randomchallenge.c	2012-05-25 09:00:27.000000000 -0400
@@ -28,6 +28,9 @@
 
 #include "opie_cfg.h"
 #include "opie.h"
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
 
 static char *algids[] = { 0, 0, 0, "sha1", "md4", "md5" };
 
diff -uNr contrib.old/opie/libopie/verify.c contrib/opie/libopie/verify.c
--- contrib.old/opie/libopie/verify.c	2012-05-25 08:54:33.000000000 -0400
+++ contrib/opie/libopie/verify.c	2012-05-25 09:12:09.000000000 -0400
@@ -25,6 +25,7 @@
 #include <string.h>
 #endif /* HAVE_STRING_H */
 #include "opie.h"
+#include <ctype.h>
 
 #define RESPONSE_STANDARD  0
 #define RESPONSE_WORD      1
diff -uNr contrib.old/opie/libopie/version.c contrib/opie/libopie/version.c
--- contrib.old/opie/libopie/version.c	2012-05-25 08:54:33.000000000 -0400
+++ contrib/opie/libopie/version.c	2012-05-25 09:13:15.000000000 -0400
@@ -19,6 +19,8 @@
 */
 #include "opie_cfg.h"
 #include "opie.h"
+#include <stdio.h>
+#include <stdlib.h>
 
 VOIDRET opieversion FUNCTION_NOARGS
 {
diff -uNr contrib.old/opie/opie.h contrib/opie/opie.h
--- contrib.old/opie/opie.h	2012-05-25 08:54:33.000000000 -0400
+++ contrib/opie/opie.h	2012-05-25 09:21:28.000000000 -0400
@@ -158,6 +158,9 @@
 int __opiereadrec __P((struct opie *));
 int __opiewriterec __P((struct opie *));
 int __opieparsechallenge __P((char *buffer, int *algorithm, int *sequence, char **seed, int *exts));
+VOIDRET opiehashlen FUNCTION((algorithm, in, out, n), int algorithm AND VOIDPTR in AND struct opie_otpkey *out AND int n);
+int opieinsecure FUNCTION_NOARGS;
+int opienewseed FUNCTION((seed), char *seed);
 __END_DECLS
 
 #define opiestrncpy(dst, src, n) \
